Web_Development/FrontEnd/HTML Links.md

```diff
@@ -0,0 +1,31 @@
+# HTML Links
+
+- Almost every web page contains one or two links atleast. These links allow users to jump from one document to another document by clicking on it.
+- We can also say that HTML links are hyperlinks.
+- When we move the mouse cursor over the link, the mouse cursor change into small hand which point over the link.
+- A link can be button, image or any other HTML elements.It does not have to be text.
+
+
+## Syntax
+
+`<a>` tag explains a hyperlink. It has the following syntax :
+
+```html
+<a href=" ">text</a>
+```
+- `href` attribute is the most important attribute of `<a>`element which designate the link's destination (mostly it is URL).
+- The _text_part is visible to reader.
+ 
+**Example**
+
+```html
+<a href="https://gwoc.girlscript.tech/">GIRLSCRIPT</a>
+```
+
+A HTML link exhibit in different colors on the basis of whether it is visited, unvisited or active.
+
+1. For Visited: links might be underlined and blue. 
+2. For Unvisited: link might be underlined and purple.
+3. For Active: link might be underlined and red.
+
+We can also create a bookmarks using HTML links.
```

Comment:
Can you add some examples for this? Tell more about Internal vs. External Links and the use of `target` attribute.

Reply:
sure